helpers.py

```diff
@@ -1,3 +1,36 @@
+import random
+import string
+import smtplib
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+
+
+def getCode(length):
+    # generate random string with length size"""
+    code = string.ascii_lowercase
+    return ''.join(random.choice(code) for i in range(length))
+
+
+def activationMail(email, username, code):
+    msg = MIMEMultipart()
+    msg['From'] = 'moviesearch@noreply.npak0382.odns.fr'
+    msg['To'] = email
+    msg['Subject'] = 'movieSearch Account activation'
+    message = """ Hello """ + username + """ 
+    
+Please confirm your account at your first login with the following code: """ + code + """
+    
+Best regards, 
+    
+Movie-Search Team
+    """
+    msg.attach(MIMEText(message))
+    mailServer = smtplib.SMTP_SSL('kapre.o2switch.net', 465)
+    mailServer.login('moviesearch@noreply.npak0382.odns.fr', 'mot2P@sseMovieSearch')
+    mailServer.sendmail('moviesearch@noreply.npak0382.odns.fr', email, msg.as_string())
```

Comment:
Je pense que pour le futur le mot de passe devrait être une variable d'environnement comme l'API secret de CS50. 
J'ai fait un exemple dans le dernier commit . 
Pour la config de ton coté : https://www.techcoil.com/blog/how-to-set-environment-variables-for-your-python-application-from-pycharm/